app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectEngagement.php

```diff
@@ -51,4 +51,9 @@ public function getTitle(): string | Htmlable
     {
         return 'Manage Prospect Email and Texts';
     }
+
+    public static function canAccess(array $parameters = []): bool
+    {
+        return auth()->user()->can('engagement.view-any');
+    }
```

Comment:
I don't think this is needed. We don't want to gate the viewing of this behind this permission Do we do this for the others?